src/selectivity-base.js

```diff
@@ -181,6 +181,8 @@ function Selectivity(options) {
 
     this.setOptions(options);
 
+    this.$el.attr('tabindex', options.tabindex || 1);
```

Comment:
Maybe we should just use the default tabindex (tabidex=0) when no tabindex is given, then the tab order would be handled by the order in sourcecode.